models/comments.js

```diff
@@ -0,0 +1,15 @@
+const db = require("../db/connection");
+
+exports.fetchCommentsByArticleId = (article_id) => {
+    return db.query(`SELECT * FROM comments WHERE article_id = $1 ORDER BY created_at DESC;`, [article_id])
+    .then(({rows}) => {
+      const comments = rows;
+      if (comments.length === 0) {
```

Comment:
How would you make sure, that the reason for getting an empty array is because there are no comments for that article, or because that article for that id doesn't exist?

In this case where the ID exists, but there are no comments, you are still sending 404 Path not found. But it should respond with an empty array and status 200.